Dockerfile.rhel7

```diff
@@ -5,8 +5,9 @@ ENV GO_PACKAGE github.com/openshift/cluster-kube-apiserver-operator
 RUN make build --warn-undefined-variables
 
 FROM registry.svc.ci.openshift.org/ocp/4.3:base
-RUN mkdir -p /usr/share/bootkube/manifests /manifests
-COPY --from=builder /go/src/github.com/openshift/cluster-kube-apiserver-operator/bindata/bootkube/* /usr/share/bootkube/manifests/
```

Comment:
@bparees out of curiosity does it mean that in the past the directory structure for `bootkube` was incorrect?

Reply:
no, you were just reliant on how "docker" handles `/*` which is not the same as how buildah does.  So it was fine as long as you were being built on docker.  As we're switching CI to build things using buildah, your dockerfile would have stopped building the right thing.

Reply:
sorry, you were reliant on imagebuilder behavior, not docker.  docker+buildah work the same way.

Reply:
see https://docs.google.com/document/d/1TJwlRj9C9HU-OhxKWnmd2O0FtYuXAZ3G0aFsD9UZPOA/edit#heading=h.tkhqbleqgny

Reply:
imagebuilder preserved the subdirs when doing a cp from `foo/*` whereas docker+buildah flatten the subdir when you use *

Reply:
okay, I was just wondering why CI jobs were passing despite the destination dir being different. Thanks for explaining!